### PR TITLE
Basic wireframes created

### PR DESCRIPTION
**Pull request by PatrickCmd**
Created basic wireframes or mockups for the bucket application
Added uml class diagrams for the application
created user account, login templates
created bucket_list templates and bucketitem templates
@patlub @CeciliaCaroline

### CODE REVIEW

**Comment by PatrickCmd** [comment]
"Added uml class diagrams for the application"

**Comment by CeciliaCaroline** [comment]
Good application templates

**Comment by patlub** [comment]
Nice UML diagrams. Good templates

File: HTML CSS Templates(Frontend)/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <!-- Fontawesome CSS -->
+    <link href="assets/font-awesome/css/font-awesome.css" rel="stylesheet">
+
+    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+    <link href="../../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
+
+    <!-- Custom styles for this template -->
+    <link href="assets/css/styles.css" rel="stylesheet">
+
+    <![endif]-->
```
Comment (skipcheru) [inline HTML CSS Templates(Frontend)/index.html]: I'm curious of the `<![endif]-->` line.What's the role of it?

**Pull request merged**